deploy/clowdapp.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ingress
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ingress
+  spec:
+    envName: ${ENV_NAME}
+    pods:
+    - name: ingress
+      image: ${IMAGE}:${IMAGE_TAG}
+      minReplicas: ${{REPLICAS}}
+      livenessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /api/ingress/v1/version
+          port: 8000
```

Comment:
should this port be 3000 here?

Reply:
clowder specifies the web port apps should use.  this means we will need to update 3scale when deploying to stage and prod.  but that means we can deploy the new app alongside the old one with an easy rollback plan.

Reply:
sweet